src/lib/utilities.py

```diff
@@ -22,6 +22,8 @@
 
 K8S_CONTAINER_NAME_PREFIX = "dynatrace-gcp-function"
 
+RELEASE_VERSION = os.environ.get("TRAVIS_TAG", "")
```

Comment:
I think the code should not know about pipeline technology we have, this is minor but would prefer to have env var also named "RELEASE_VERSION"